geoips/image_utils/mpl_utils.py

```diff
@@ -706,7 +717,10 @@ def create_colorbar(fig, mpl_colors_info):
 
     # Allow arbitrary kwargs to set_label
     set_label_kwargs = {}
-    if "set_label_kwargs" in mpl_colors_info:
+    if (
```

Comment:
Just thinking. I do this too, but I wonder if this kind of thing would be better written as:
```
if mpl_colors_info.get('set_label_kwargs', None):
```

Reply:
@jsolbrig I updated these merged to update_products - data_fusion passed, and geoips repo tests are currently passing. Please resolve if you are good with this.

Reply:
Looks good. Thank you!